Soundex.test.py

```diff
@@ -1,14 +1,16 @@
 import unittest
 from Soundex import generate_soundex
-
 class TestSoundex(unittest.TestCase):
-
     def test_empty_string(self):
         self.assertEqual(generate_soundex(""), "")
-
     def test_single_character(self):
         self.assertEqual(generate_soundex("A"), "A000")
-
-    
+    def test_simple_cases(self):
```

Comment:
Unit Test code is considered as a low level design  document ! it better to give semantic name for testcase